api/server/structs/conversions.go

```diff
@@ -1074,3 +1074,17 @@ func sszBytesToUint256String(b []byte) (string, error) {
 	}
 	return bi.String(), nil
 }
+
+func DepositSnapshotFromConsensus(ds *eth.DepositSnapshot) *DepositSnapshot {
```

Comment:
Please add tests for this.

Reply:
Yes, that is true. I think there is a gap on the finalized slice. 

There is also some API constraints that mention this should be exactly three items. 
Do we enforce min/max? If not, should we?

```yaml
  properties:
    finalized:
      type: array
      items:
        $ref: './primitive.yaml#/Root'
      minItems: 3
      maxItems: 3
 ```

Reply:
I added a test so that we are also checking that the finalized slice matches.
None of the other conversions have tests should we consider adding them at some point?

Reply:
I think that would be a lot of work for a small gain